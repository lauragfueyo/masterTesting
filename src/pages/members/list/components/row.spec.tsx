import * as React from 'react';
import { shallow } from 'enzyme';
import { Member } from '../viewModel';
import { Row } from './row';

describe('pages/members/list/components/row tests', () => {
    it('should render as expected when passing one member', () => {
        // Arrange
        const props = {
            id: 1,
            name: 'test name',
            avatarUrl: 'test url',
        };
        
  
        // Act
        const component = shallow(
            <Row member={props} />
        );
  
        // Assert
        expect(component).toMatchSnapshot();
   });
});