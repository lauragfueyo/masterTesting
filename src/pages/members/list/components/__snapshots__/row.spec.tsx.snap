// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pages/members/list/components/row tests should render as expected when passing one member 1`] = `
<tr>
  <td>
    <img
      src="test url"
      style={
        Object {
          "maxWidth": "10rem",
        }
      }
    />
  </td>
  <td>
    <span>
      1
    </span>
  </td>
  <td>
    <span>
      test name
    </span>
  </td>
</tr>
`;

exports[`pages/members/list/components/row tests should render as expected when passing one member properties 1`] = `
<tr>
  <td>
    <img
      src="test url"
      style={
        Object {
          "maxWidth": "10rem",
        }
      }
    />
  </td>
  <td>
    <span>
      1
    </span>
  </td>
  <td>
    <span>
      test name
    </span>
  </td>
</tr>
`;

exports[`pages/members/list/components/row tests should render as expected when passing required properties 1`] = `
<tr>
  <td>
    <img
      src="test url"
      style={
        Object {
          "maxWidth": "10rem",
        }
      }
    />
  </td>
  <td>
    <span>
      1
    </span>
  </td>
  <td>
    <span>
      test name
    </span>
  </td>
</tr>
`;
